imagetool/src/main/java/com/oracle/weblogic/imagetool/cli/menu/CreateAuxImage.java

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2021, Oracle and/or its affiliates.
+// Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+package com.oracle.weblogic.imagetool.cli.menu;
+
+import java.io.File;
+import java.time.Instant;
+import java.util.concurrent.Callable;
+
+import com.oracle.weblogic.imagetool.api.model.CommandResponse;
+import com.oracle.weblogic.imagetool.logging.LoggingFacade;
+import com.oracle.weblogic.imagetool.logging.LoggingFactory;
+import com.oracle.weblogic.imagetool.util.Utils;
+import picocli.CommandLine.ArgGroup;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.IDefaultValueProvider;
+import picocli.CommandLine.Model.ArgSpec;
+
+import static com.oracle.weblogic.imagetool.util.Constants.BUSYBOX;
+
+@Command(
+    name = "createAuxImage",
+    description = "Build Auxiliary docker image for WebLogic domain configuration",
+    requiredOptionMarker = '*',
+    defaultValueProvider = CreateAuxImage.class,
+    abbreviateSynopsis = true
+)
+public class CreateAuxImage extends CommonOptions
+    implements Callable<CommandResponse>, IDefaultValueProvider {
+    private static final LoggingFacade logger = LoggingFactory.getLogger(CreateAuxImage.class);
+
+    @Override
+    public CommandResponse call() throws Exception {
+        Instant startTime = Instant.now();
+
+        try {
+            initializeOptions();
+
+            // WDT install and artifacts should be defaulted to the /auxiliary directory instead of /u01/wdt
+            dockerfileOptions.setWdtHome("/auxiliary");
```

Comment:
Should this be hard coded?

Reply:
Is there any reason that this needs to be configurable? I don't yet see any reason why the customer would care about this choice.

Reply:
There is a --wdtHome  option in cli help 

WDT Options
      --wdtArchive=<wdtArchivePath>
                            A WDT archive zip file, if needed (or
                              comma-separated list of files).
      --wdtHome=<wdtHome>   The target folder in the image for the WDT install
                              and models. Default: /u01/wdt

Reply:
We can update the cli help stating this will be ignored and default to /auxiilary in createAuxImage if it is hard to refactor out the wdtHome in options.

Reply:
Why not?